.github/README.md

```diff
@@ -0,0 +1,130 @@
+```
+Perl is Copyright (C) 1993, 1994, 1995, 1996, 1997, 1998, 1999, 2000,
+2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009, 2010, 2011, 2012,
+2013, 2014, 2015, 2016, 2017, 2018, 2019, 2020, 2021 by Larry Wall and others.
+All rights reserved.
+```
+
+
+# About Perl
+
+Perl is a general-purpose programming language originally developed for
+text manipulation and now used for a wide range of tasks including
+system administration, web development, network programming, GUI
+development, and more.
+
+The language is intended to be practical (easy to use, efficient,
+complete) rather than beautiful (tiny, elegant, minimal).  Its major
+features are that it's easy to use, supports both procedural and
+object-oriented (OO) programming, has powerful built-in support for text
+processing, and has one of the world's most impressive collections of
+third-party modules.
+
+For an introduction to the language's features, see [pod/perlintro.pod](https://perldoc.perl.org/perlhack).
+
```

Comment:
As this is (now) the "README for github" file, I think that all these documentation links should point to github instead of perldoc.perl.org

Hence

    For an introduction to the language's features, see [pod/perlintro.pod](https://github.com/Perl/perl5/blob/blead/pod/perlhack.pod).

github renders the pod nicely, and in checking that, I realised that it actually makes **more** sense, given what `perlhack.pod` says:

```
=item * Ensure you're following the latest advice

In case the advice in this guide has been updated recently, read the
latest version directly from the perl source:

  % perldoc pod/perlhack.pod
```

perldoc.perl.org is only showing the version of the file from the most recent stable release, so might well be out of date. The blead blob is, well, the bleading edge.

Reply:
I agree, based on `perlhack` I'm going to modify URLs to link to more recent versions.